src/_events/hacknotts-2018/description.md

```diff
@@ -0,0 +1,23 @@
+## What's a hackathon?
+
+A [hackathon](https://en.wikipedia.org/wiki/Hackathon) is a collaborative programming festival. The main goal is to try to create a project in a relatively short amount of time, such as 24 hours. It's a fantastic opportunity to meet new people, learn from mentors, and make cool stuff.
+
+## What's HackNotts?
+
+HackNotts is our annual hackathon, hosted in the School of Computer Science on Jubilee Campus. This year, we'the event will be running over the weekend of the 24th and 25th of November, with the opening presentations starting some time around 11am on the Saturday, and the closing presentation will be early in the evening on the Sunday.
+
+## Who can attend?
+
+You're allowed to attend HackNotts if you are:
+
+- A high school student,

```

Comment:
This is not confirmed yet. Lots of hoops to jump through.

Reply:
Perhaps change to 18 years or older

Reply:
That's fair, I was basing my description on the FAQ from a past event, so details have likely changed. If we're not sure yet, would it be better to just remove this line, seeing as it's covered already by the university student lines following it?

Reply:
I think 18 on It's own. And then High School, Undergrad, and Postgrad

Reply:
Sounds good to me!